src/usts/ewma.cpp

```diff
@@ -39,7 +33,13 @@ auto ewma<Clock>::add(time_point time, double value) -> void {
 
 template<typename Clock>
 auto ewma<Clock>::get() const -> double {
-    return average;
+    return get(clock_type::now());
+}
+
+template<typename Clock>
+auto ewma<Clock>::get(time_point time) const -> double {
+    const auto alpha = calculate_alpha(time);
```

Comment:
Why have you parameterized already processed (finalized) in `add` time result by time parameter again? It seems  that it is just the retriever method by concept, and clients could be aware only on this behavior.